sympy/series/series.py

```diff
@@ -4,9 +4,55 @@
 
 
 def series(expr, x=None, x0=0, n=6, dir="+"):
-    """Series expansion of expr around point `x = x0`.
+    """Returns
+       =======
+       Series expansion of expr around point `x = x0`.
 
-    See the docstring of Expr.series() for complete details of this wrapper.
+       Parameters
+       ==========
+
+       expr : Expression
+              The expression whose series is to be expanded.
+
+       x    : Symbol
+              It is the variable of the expression to be calculated.
+
+       x0   : Value
+              The value around which ``x`` is calculated. Can be ``oo`` or
```

Comment:
Can it only be `-oo` or `oo`?
Your examples say otherwise.
Did you mean it can be between `-oo to oo` ?

Reply:
Oh! yes My bad. I am changing it.